src/styles/blocks/card.scss

```
@@ -0,0 +1,136 @@
+// use nprmalize scss
+body {
+  margin: 0;
+  padding: 0;
+
+  font-weight: 400;
+  line-height: 14px;
+}
+
+h3 {
+  font-weight: 500;
+}
+
+.card {
+  box-sizing: border-box;
+
+  height: 408px;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+
+  border-radius: 5px;
+  background-color: $primary-bg-color;
+  border: 1px solid $primary-bg-color;
+
+  &__image{
+    margin: 0;
+    padding: 32px 20px 0;
+    height: 134px;
+    width: 160px;
+
+    object-fit: contain;
+    object-position: center;
+    align-self: center;
+  }
+
+  &__product-name {
+    margin: 0;
+    padding: 40px 16px 4px;
+    height: 36px;
+
+    text-align: start;
+    line-height: 18px;
+    font-weight: 500px;
+    font-size: 12px;
+    color: $main-text-color;
+
+    transition-duration: $transition-duration;
```

don't use it like that, because it's the same as transition: all 0.3s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md

Create a variable for transition duration
For example
```suggestion
    transition: color $transition-duration, transform $transition-duration;
```